src/widgets/toggle-selector.ts

```diff
@@ -0,0 +1,35 @@
+import { Widget } from './widget';
+import { ElementArrayFinder } from '../wdio';
+
+export interface ToggleSelectorOption {
+    id: string;
+    caption: string;
+}
+
+export class ToggleSelector extends Widget {
+
+    public async selectOptionById(optionId: string) {
+        await this.byId(optionId).click();
+    }
+
+    public async getSelectedOption(): Promise<string> {
+        return this.byClassName('slab-option slab-selected ng-star-inserted').getText();
```

Comment:
Do we really need to include "ng-star-inserted" in the class selector? Can't this be done with an expression that better facilitates identifying the element with the option text?

Reply:
I am not sure. At that moment it was the only/quick way I found. It could be improved

Reply:
I have uploaded ToggleSelector WO's code as it was recently reshaped by @joaquimvila for the project.
Thus a brand new review is needed